gpflow/models/sgpmc.py

```diff
@@ -86,7 +87,13 @@ def __init__(
             loc=to_default_float(0.0), scale=to_default_float(1.0)
         )
 
-    def log_likelihood(self, *args, **kwargs) -> tf.Tensor:
+    def log_posterior_density(self) -> tf.Tensor:
+        return self.log_conditional_likelihood_lower_bound() + self.log_prior_density()
+
+    def maximum_log_likelihood_objective(self) -> tf.Tensor:
+        return self.log_conditional_likelihood_lower_bound()
+
+    def log_conditional_likelihood_lower_bound(self) -> tf.Tensor:
```

Comment:
@jameshensman if you have a better suggestion for how to name this, please shout!